package.json

```diff
@@ -111,16 +111,19 @@
     "@material/theme": "^1.1.0",
     "@storybook/addon-viewport": "^5.0.11",
     "@storybook/react": "^5.0.11",
+    "@wordpress/a11y": "^2.2.0",
     "@wordpress/api-fetch": "^2.2.8",
     "@wordpress/babel-preset-default": "^4.3.0",
     "@wordpress/browserslist-config": "^2.3.0",
-    "@wordpress/components": "^7.2.0",
+    "@wordpress/compose": "^3.2.0",
+    "@wordpress/dom": "^2.2.0",
```

Comment:
Why do we need to add these packages (meaning all except `compose` which is clear to me)? I don't see them used anywhere.

Reply:
Definitely helps to reflect what kind of packages are being used by the application. Especially with #437 this is more useful as you get all the benefits like dependency resolution and IDE autocompletion.

Reply:
When I removed `@wordpress/components` the webpack build started failing with these errors:
```
ERROR in Entry module not found: Error: Can't resolve './node_modules/@wordpress/a11y' in '/Users/aaemnnosttv/10up/Sites/googlesitekit-test/wordpress/wp-content/plugins/google-site-kit'

ERROR in Entry module not found: Error: Can't resolve './node_modules/@wordpress/compose' in '/Users/aaemnnosttv/10up/Sites/googlesitekit-test/wordpress/wp-content/plugins/google-site-kit'

ERROR in Entry module not found: Error: Can't resolve './node_modules/@wordpress/dom' in '/Users/aaemnnosttv/10up/Sites/googlesitekit-test/wordpress/wp-content/plugins/google-site-kit'

ERROR in Entry module not found: Error: Can't resolve './node_modules/@wordpress/keycodes' in '/Users/aaemnnosttv/10up/Sites/googlesitekit-test/wordpress/wp-content/plugins/google-site-kit'
```

These are dependencies that were installed by `@wordpress/components` (all referenced under `externalPackages` in the webpack.config but weren't defined as explicit dependencies.

Reply:
@swissspidy @aaemnnosttv Yes, I was thinking along these lines too, but where are these dependencies even used? I don't see any reference in our JS code, and they also don't seem to be dependencies of our dependencies, so I'm not sure.

Reply:
They are all

> These are dependencies that were installed by @wordpress/components (all referenced under externalPackages in the webpack.config but weren't defined as explicit dependencies.

Now that we don't have this dependency anymore, why do we need its sub-dependencies? Seems like some leftover code—or is the build failing when you remove those?

Reply:
The build was failing due to these packages still being defined in the webpack config's entries, so it was trying to generate a file for them the same way we polyfill other wp modules. These weren't used anymore though so I have removed them, and their entries 👍 

I see we also have a similar addition for `wp-sanitize` which is also inlined in the `post-searcher.js` but doesn't appear to be used anywhere. I think this is leftover code that can probably be removed as well here but I want to double check first.

https://github.com/google/site-kit-wp/blob/c0c9216f4708bd1b208652c152e0cd01857e458c/assets/js/components/post-searcher.js#L39-L83

Reply:
After checking with @adamsilverstein we confirmed that this shim is no longer used/needed so this has been removed as well! 😄